packages/charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -0,0 +1,186 @@
+import * as React from 'react';
+import { max as d3Max } from 'd3-array';
+import { axisLeft as d3AxisLeft, axisBottom as d3AxisBottom, Axis as D3Axis } from 'd3-axis';
+import { scaleBand as d3ScaleBand, scaleLinear as d3ScaleLinear, ScaleLinear as D3ScaleLinear } from 'd3-scale';
+import { select as d3Select } from 'd3-selection';
+import { classNamesFunction, customizable, IClassNames } from '../../Utilities';
+import {
+  IHorizontalBarChartProps,
+  IHorizontalBarChartStyleProps,
+  IHorizontalBarChartStyles,
+  IDataPoint
+} from './HorizontalBarChart.types';
+
+const getClassNames = classNamesFunction<IHorizontalBarChartStyleProps, IHorizontalBarChartStyles>();
+type numericAxis = D3Axis<number | { valueOf(): number }>;
+type stringAxis = D3Axis<string>;
+
+@customizable('HorizontalBarChart', ['theme', 'styles'])
```

Comment:
This decorator is no longer needed (the `styled` helper that you're using in `HorizontalBarChart.tsx` replaces it)